Packs/CommonPlaybooks/Playbooks/playbook-Dedup_-_Generic_v4.yml

```diff
@@ -733,8 +772,7 @@ tasks:
                 - - operator: greaterThanOrEqual
                     left:
                       value:
-                        simple: DBotFindSimilarIncidentsByIndicators.similarIncident.similarity
-                          indicators
+                        simple: DBotFindSimilarIncidentsByIndicators.similarIncident.similarity indicators
```

Comment:
There is a space between similarity and indicators.
As far as I know there are no spaces in context paths.

Reply:
the context path includes a space between similarity and indicators.
<img width="1508" alt="Screenshot 2023-10-16 at 21 52 34" src="https://github.com/demisto/content/assets/112805149/849790bb-0346-4cc7-8426-3fe6f0afe3b6">